src/dao/dynamo/dao.ts

```diff
@@ -1,7 +1,7 @@
-import { isObject } from '../object'
+import { isObject } from '../../object'
 import dbClient from 'aws-dynamodb-factory-js'
-import { error } from '../error'
-import type { DynamodbParams, DynamodbResponseBatch } from './interfaces'
+import { error } from '../../error'
+import type { DynamodbParams, DynamodbResponseBatch } from '../interfaces'
```

Comment:
I think you can move dynamo interfaces to dynamo/types as well

Reply:
Sure. I'll do it!